packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf

```diff
@@ -294,6 +296,159 @@ dbms.directories.import=import
 # true, it will never be elected as master.
 #ha.slave_only=false
 
+#********************************************************************
+# Security Configuration
+#********************************************************************
+
+# The authentication and authorization provider that contains both users and roles.
+# This can be one of the built-in `native` or `ldap` auth providers,
+# or it can be an externally provided plugin, with a custom name prefixed by `plugin`,
+# i.e. `plugin-<AUTH_PROVIDER_NAME>`.
+#dbms.security.auth_provider=native
+
+#================================================
+# LDAP Auth Provider Configuration
+#================================================
+
+# URL of LDAP server (with protocol, hostname and port) to use for authentication and authorization.
+# If no protocol is specified the default will be `ldap://`.
+# To use LDAPS, set the protocol and port, e.g. `ldaps://ldap.example.com:636`
+# NOTE: You may want to consider using STARTTLS (`dbms.security.ldap.use_starttls`) over LDAPS
+# for secure connections, in which case the default port will be sufficient.
+#dbms.security.ldap.host=0.0.0.0:389
```

Comment:
This doesn't make much sense as a default. `0.0.0.0` is a wild card for all interfaces when binding to them. But it makes no sense when connecting to something.

So if anything, it should be `localhost`